### PR TITLE
Add postlink scripts

### PR DESCRIPTION
**Pull request by geof90**
This PR adds postlink scripts that RNPM will automatically call when executing `react-native link react-native-code-push` or `rnpm link react-native-code-push` so as to simplify setup.

Once https://github.com/facebook/react-native/pull/9157 goes through, we can also add `postunlink` scripts as well to simplify removal of our plugin.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
     "url": "https://github.com/Microsoft/react-native-code-push"
   },
   "dependencies": {
-    "code-push": "1.8.0-beta"
+    "code-push": "1.8.0-beta",
+    "inquirer": "^1.1.2",
+    "plist": "^1.2.0"
```
Comment (silhouettes) [inline package.json]: Should we pin these dependencies too?

File: README.md [hunk 2]
```diff
@@ -110,6 +114,8 @@ In order to accommodate as many developer preferences as possible, the CodePush
 
     *Note: If you don't already have RNPM installed, you can do so by simply running `npm i -g rnpm` and then executing the above command. If you already have RNPM installed, make sure you have v1.9.0+ in order to benefit from this one step install.*
 
+2. You will be prompted for the deployment key you'd like to use. If you don't already have it, you can retreive this value by running `code-push deployment ls <appName> -k`, or you can choose to ignore it (by simply hitting `<ENTER>`) and add it in later. To get started, we would recommend just using your `Staging` deployment key, so that you can test out the CodePush end-to-end.
```
Comment (silhouettes) [inline README.md]: retrieve

**Comment by silhouettes** [comment]
Would it be worth doing something to check that the app version is semver and/or allow them to change it?

**Comment by geof90** [comment]
> Would it be worth doing something to check that the app version is semver and/or allow them to change it?

I'm hesitant.. mainly a) I don't see that as part of linking and b) not everyone uses RNPM/react-native link so its not going to help everyone. If anything, I feel like we could display a warning when they use `release-react`.

**Pull request merged**